src/declarations/stencil-private.ts

```diff
@@ -197,10 +197,16 @@ export interface RollupResults {
 export interface BuildCtx {
   buildId: number;
   buildResults: CompilerBuildResults;
+  buildStats?: CompilerBuildStats;
   buildMessages: string[];
   bundleBuildCount: number;
   collections: Collection[];
   compilerCtx: CompilerCtx;
+  esmBrowserComponentBundle: BundleModule[];
+  esmComponentBundle: BundleModule[];
+  es5ComponentBundle: BundleModule[];
+  systemComponentBundle: BundleModule[];
+  commonJsComponentBundle: BundleModule[];
```

Comment:
I was worried that adding these to the build context would increase the memory usage significantly. Luckily that is not the case. I ran 10 builds of ionic-framework using `master` and this PR. The increase is relatively minor:

| Build #     | `master`         | #3030            |
| ----------- | ---------------: | ---------------: |
| 1           | MEM: 682.1MB     | MEM: 658.6MB     |
| 2           | MEM: 743.8MB     | MEM: 677.3MB     |
| 3           | MEM: 546.5MB     | MEM: 757.6MB     |
| 4           | MEM: 727.2MB     | MEM: 697.9MB     |
| 5           | MEM: 731.8MB     | MEM: 658.1MB     |
| 6           | MEM: 730.2MB     | MEM: 656.2MB     |
| 7           | MEM: 738.5MB     | MEM: 749.1MB     |
| 8           | MEM: 726.2MB     | MEM: 655.7MB     |
| 9           | MEM: 709.2MB     | MEM: 684.9MB     |
| 10          | MEM: 663.2MB     | MEM: 617.7MB     |
| **Average** | **MEM: 681.3MB** | **MEM: 699.9MB** |

Reply:
@ltm is there a process you used for this that we could doc/reproduce in the future? I could see this also being useful for build times as well